prusti-tests/tests/parse/ui/foreign_mods.stderr

```diff
@@ -0,0 +1,42 @@
+  __          __        __  ___             
+ |__)  _\/_  |__) |  | /__`  |   ____\/_  | 
+ |      /\   |  \ \__/ .__/  |       /\   | 
+
+Prusti version: 0.2.1, commit 34981398a0b 2022-11-08 21:01:25 UTC, built on 2022-11-10 12:18:31 UTC
+error[E0308]: mismatched types
+ --> prusti-tests/tests/parse/ui/foreign_mods.rs:5:26
```

Comment:
Ah right (seeing the CI failure), the paths here need to be changed to `$DIR/foreign_mods.rs`.

Reply:
Should be fixed.